server/src/api/controllers/sessionController.js

```diff
@@ -45,20 +46,22 @@ export async function createSession(req, res) {
   }
 
   try {
-    const progress = quest.stages[0].id
     const container = await createContainer(
       `quest-${quest.id}-${req.user.username}-${Date.now()}`
     )
     const newSession = new Session({
       user: req.user,
       quest,
-      progress,
       containerId: container.id,
     })
 
-    const session = await newSession.save()
-    res.status(201).json(session)
+    const session = new SessionHandler(newSession)
+    session.addNewTasks()
```

Comment:
The progress is added in addNewTasks, right

Reply:
That's right